### PR TITLE
Adding "About" and "Contact" pages

### PR DESCRIPTION
**Pull request by SolomiiaLomnytska22**
**Refs:**
#11 (Add more pages)

**Proposed changes:**
- Added "About" page.
- Added "Contact" page.
- Added those pages to the router.ts.

User interface before implementing changes: 
https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/b23fc0ea-2069-4a3a-9a18-4d7f43548907

User interface after implementing changes:
Screenshot 2023-12-06 at 09 13 11
Screenshot 2023-12-06 at 09 13 42

**Verification:**
To see "About" page:
- Run the project.
- Click on the side menu icon.
- Select "About" page from the menu.

To see "Contact" page:
- Run the project.
- Click on the side menu icon.
- Select "Contact" page from the menu.

All changes tested locally. New and already existing functionality is supported.

https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/5876103e-7eee-498c-b04c-42908c59e42e

### CODE REVIEW

File: app/src/About.vue [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+    <ScrollableCards :items="items" />
+
+    <section class="timeline">
+      <h2>Our Journey</h2>
+      <Timeline />
+    </section>
+
+    <h1>Let's build our future together!</h1>
+    <router-link to="/contact">
+      <Button><p>Contact Us</p></Button>
```
Comment (marina-grechukha) [inline app/src/About.vue]: do you really need `<p>` inside `<Button>`?

File: app/src/Contact.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        <a
+          href="https://www.twitter.com"
+          target="_blank"
+        >
+          <font-awesome-icon
+            size="xl"
+            icon="fa-brands fa-x-twitter"
+          />
+        </a>
+      </div>
```
Comment (marina-grechukha) [inline app/src/Contact.vue]: looks like this could be `v-for`

File: app/src/components/about/ScrollableCards.vue [hunk 3]
```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator'
+
+@Component({})
+export default class ScrollableCards extends Vue {
+  @Prop({ required: true }) items!: {
+    id: number
+    image: string
+    title: string
+    description: string
```
Comment (marina-grechukha) [inline app/src/components/about/ScrollableCards.vue]: this could be a type

File: app/src/About.vue [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+import Timeline from '@/components/about/Timeline.vue'
+import ScrollableCards from '@/components/about/ScrollableCards.vue'
+import Button from '@/components/common/Button.vue'
+
+@Component({ components: { Button, Timeline, ScrollableCards } })
+export default class About extends Vue {
+  items = [
```
Comment (kaytee-te) [inline app/src/About.vue]: Missed type for `items`.

File: app/src/About.vue (hunk 1 above)
Comment (SolomiiaLomnytska22) [reply app/src/About.vue]: Thanks, just removed it in my new commit!

File: app/src/Contact.vue (hunk 2 above)
Comment (SolomiiaLomnytska22) [reply app/src/Contact.vue]: Fixed!

File: app/src/components/about/ScrollableCards.vue (hunk 3 above)
Comment (SolomiiaLomnytska22) [reply app/src/components/about/ScrollableCards.vue]: Added new type in the new commit.

File: app/src/About.vue (hunk 4 above)
Comment (SolomiiaLomnytska22) [reply app/src/About.vue]: Added the type in the new commit!

File: app/src/Contact.vue [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+
+@Component({})
+export default class Contact extends Vue {
+  socialLinks = [
```
Comment (marina-grechukha) [inline app/src/Contact.vue]: type?

File: app/src/components/about/Timeline.vue [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component } from 'vue-property-decorator'
+
+@Component({})
+export default class Timeline extends Vue {
+  timeline = [
```
Comment (marina-grechukha) [inline app/src/components/about/Timeline.vue]: type?

File: app/src/types.ts [hunk 7]
```diff
@@ -13,5 +13,11 @@ interface MenuItem {
   route: string
   title: string
 }
+interface CardsItem{
```
Comment (marina-grechukha) [inline app/src/types.ts]:
```suggestion
interface CardItem{
```

File: app/src/components/about/Timeline.vue [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+    {
+      id: 5,
+      date: 'Present',
+      title: 'Continued Growth',
+      description: 'Continuing to evolve and grow with our customers.'
+    }
+  ]
+
+  getStyleType (id: number): string {
+    return id % 2 === 0 ? 'left' : 'right'
```
Comment (kaytee-te) [inline app/src/components/about/Timeline.vue:60]: Please comment about a magic number.

File: app/src/Contact.vue (hunk 5 above)
Comment (SolomiiaLomnytska22) [reply app/src/Contact.vue]: Added type in recent commit.

File: app/src/components/about/Timeline.vue (hunk 6 above)
Comment (SolomiiaLomnytska22) [reply app/src/components/about/Timeline.vue]: Added type in recent commit.

File: app/src/types.ts (hunk 7 above)
Comment (SolomiiaLomnytska22) [reply app/src/types.ts]: Fixed

File: app/src/components/about/Timeline.vue (hunk 8 above)
Comment (SolomiiaLomnytska22) [reply app/src/components/about/Timeline.vue:60]: Added comment in recent commit

**Review by marina-grechukha — APPROVED** [review]

**Review by kaytee-te — APPROVED** [review]

**Pull request merged**